doc/api/process.markdown

```diff
@@ -70,6 +70,16 @@ Example of listening for `exit`:
     });
 
 
+## Event: 'message'
+
+* `message` {Object} a parsed JSON object or primitive value
+* `sendHandle` {Handle object} a Socket or Server object
```

Comment:
good point - this is cut-n-paste from child_process docs, I'll fixup both places